StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory+Boleto.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  PaymentSheetFormFactory+Boleto.swift
+//  StripePaymentSheet
+//
+//  Created by Eduardo Urias on 9/11/23.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+@_spi(STP) import StripeUICore
+
+extension PaymentSheetFormFactory {
+    func makeBoleto() -> FormElement {
+        let contactInfoSection = makeContactInformationSection(
+            nameRequiredByPaymentMethod: true,
+            emailRequiredByPaymentMethod: true,
+            phoneRequiredByPaymentMethod: false
+        )
+        let taxIdElement = TextFieldElement(
+            configuration: IDNumberTextFieldConfiguration(
+                type: .BR_CPF_CNPJ,
+                label: STPLocalizedString("CPF/CPNJ", "Label for CPF/CPNJ (Brazil tax ID) field")
```

Comment:
nit: preferred pattern is to add to `String+Localized.swift` in the PaymentSheet module